### ISSUE TITLE
dls not working on ES5

### ISSUE THREAD

**Issue by djtecha** [floragunncom/search-guard-module-dlsfls#3]
I'm testing ES5 and cannot get the DLS plugin to work properly. I have installed the jar to /usr/share/elasticsearch/plugins/search-guard-5/dlic-search-guard-module-dlsfls-5.2-5.jar running the sgadmin tool I see:

***************************************************
Searchguard DLS/FLS(+) Security is not free software
for commercial use in production.
You have to obtain a license if you 
use it in production.
(+) Document-/Fieldlevel

and on startup of ES I see a lot of [2017-02-27T13:56:27,413][INFO ][c.f.s.SearchGuardPlugin  ] FLS/DLS enabled lines.

But the user logged in with the sg_public role attached is still not limited to events with the DLS attached.

**sg_roles.yml**
```
sg_admin:
  cluster:
    - '*'
  indices:
    '*':
      '*':
        - '*'
        
sg_public:
  cluster:
    - CLUSTER_MONITOR
    - MONITOR
  indices:
    '/^\w+-\S*/': 
      '*':
        - READ
        - indices:admin/mappings/fields/get*
        - indices:admin/validate/query*
        - indices:admin/get*
      _dls_: '{ "term" : {"tags" : "devlogs" } }'
    'elastalert':
      '*':
        - ALL
    '?kibana':
      '*':
        - READ
        - indices:admin/mappings/fields/get*
      _dls_: '{ "term" : {"tags" : "devlogs" } }'

sg_kibana5_server:
  cluster:
    - cluster:monitor/nodes/info
    - cluster:monitor/health
    - cluster:monitor/main
  indices:
    '*':
      '*':
        - KIBANA_USER
        - KIBANA_SERVER
    '?kibana':
      '*':
        - KIBANA_USER
        - KIBANA_SERVER
        
sg_readonly_and_monitor:
`  cluster:
    - CLUSTER_MONITOR
    - MONITOR
  indices:
    '*':
      '*':
        - MONITOR
    'elastalert':
      '*':
        - ALL

sg_logstash:
  cluster:
    - indices:admin/template/get
    - indices:admin/template/put
    - indices:data/write/bulk*  
    - cluster:monitor/main
  indices:
    '*':
      '*':
        - indices:admin/mapping/put*
        - CRUD
        - CREATE_INDEX

sg_curator:
  cluster:
    - MONITOR
    - SNAPSHOT
    - CLUSTER_MONITOR
  indices:
    '*':
      '*':
        - MANAGE_ALIASES
        - MONITOR
        - indices:admin/delete
        - indices:admin/exists
        - indices:admin/optimize
        - indices:admin/get*
        - indices:admin/settings/update 
```
**sg_roles_mapping.ym**l
```
sg_admin:
  users:
    - admin

sg_logstash:
  users:
    - logstash

sg_kibana5_server:
  users:
    - kibana


sg_public:
  users:
    - '/.*domain.com/'
    - elastalert-dev

sg_curator:
  users:
    - curator

sg_readonly_and_monitor:
  users:
    - nagios
    - elastalert-ops

sg_listing:
  users:
    - listing
```

**Comment by djtecha** [comment]
NVM. Looks like this was fixed in dlic-search-guard-module-dlsfls-5.2-6 though the download link is still pointing at dlic-search-guard-module-dlsfls-5.2-5

**Comment by floragunncom** [comment]
links updated, thx
